modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/IgniteSnapshotManager.java

```diff
@@ -394,6 +403,13 @@ public class IgniteSnapshotManager extends GridCacheSharedManagerAdapter
     /** Snapshot transfer rate limit in bytes/sec. */
     private final DistributedLongProperty snapshotTransferRate = detachedLongProperty(SNAPSHOT_TRANSFER_RATE_DMS_KEY);
 
+    /**
+     * Snapshot delta sort batch size in pages count. If set then delta pages will be indexed by page index to almost
+     * sequential disk write on apply. This generates an extra delta read. If value is non-positive or not set then delta
+     * pages will be applied directly.
+     */
+    private final DistributedIntegerProperty deltaSortBatch = detachedIntegerProperty(SNAPSHOT_DELTA_SORT_BATCH_SIZE_KEY);
```

Comment:
I'd rather avoid exposing some of the snapshot internal process things like "writing delta" or so on to a user.

Actually, I think it would be more friendly for a user to have a snapshot configuration property of a snapshot write mode `RANDOM` or `SEQUENTIAL`. The batch must be predefined by the system.

We don't even need to change this parameter at runtime.

Reply:
> We don't even need to change this parameter at runtime.

The snapshot path is set on each snapshot create process. I think we should allow users to set a mode per hardware. WDYT?

Reply:
I have refactored property to a sequential flag: `snapshotSequentialWrite`.